BetSystem/Contract/IdRequestDto.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace BetSystem.Contract
+{
+    public class IdRequestDto
```

Comment:
Dont understand this DTO - if it transfers Id do we need additional wrapper for one property?

Reply:
Its needed in some places because for exampl if i want to AddTeamToSportEvent I dont want to give name of it but only Id